zebra-consensus/src/transaction/check.rs

```diff
@@ -298,8 +307,12 @@ pub fn coinbase_expiry_height(
         // If Nu5 does not have a height, apply the pre-Nu5 rule.
         None => validate_expiry_height_max(expiry_height, true, block_height, coinbase),
         Some(activation_height) => {
-            // Consensus rule: from NU5 activation, the nExpiryHeight field of a
-            // coinbase transaction MUST be set equal to the block height.
+            // # Consensus
+            //
+            // > [NU5 onward] The nExpiryHeight field of a coinbase transaction MUST be equal
+            // > to its block height.
+            //
+            // https://zips.z.cash/protocol/protocol.pdf#txnconsensus
```

Comment:
By what i can see #3401 already has 2 conflicts in the same file, so i dont think it will make sense to base anything on that PR right now. On the contrary, it seems #3401 will need to fix conflicts to be up to date. We can just remove the change and leave #3401 handle it or keep it as it is and have #3401 deal with it as well.

Reply:
I'm happy to do whatever you think is best here.